nativeauthenticator/handlers.py

```diff
@@ -295,30 +326,42 @@ async def get(self, user_name):
         if not self.authenticator.user_exists(user_name):
             raise web.HTTPError(404)
         html = await self.render_template(
-            "change-password.html",
+            "change-password-admin.html",
             user_name=user_name,
         )
         self.finish(html)
 
     @admin_users_scope
     async def post(self, user_name):
-        new_password = self.get_body_argument("password", strip=False)
-        success = self.authenticator.change_password(user_name, new_password)
+        new_password = self.get_body_argument("new_password", strip=False)
+        confirmation = self.get_body_argument("new_password_confirmation", strip=False)
```

Comment:
I appreciate when the same data is referred to in the same way in different context even though it has been piped around, but this is acceptable to me.